content/how-to/rackspace-fabric/common-request-templates/index.md

```diff
@@ -14,35 +14,35 @@ product_url: rackspace-fabric
-| Create new VPN User on Firewall | Change | Firewall Platform | One |
-| Request Emergency Financial Accomodations | Account / Account Management | None | None |
-| Delete Contact | Account / User Management | None | None |
-| Remove Virtual IP Address from Load Balancer | Change | Load Balancer | One |
-| Generate Certificate Signing Request (CSR) | Change | Firewall Platform, Firewall Platform WAF | One |  
-| Hypervisor ESXi Version Check | Change | Hypervisor ESXi | One |
-| IP Allocation | Change | Any | One |
-| Local User Management | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |
-| Logical Security Testing | Account / User Management | None | None |
-| Login Access to Server | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |
-| Network Troubleshooting | Change | Firewall Platform, Firewall Platform WAF, Hypervisor, Hypervisor ESXi, Load Balancer, Managed Shared Storage, RPC-V, Server, VM | One |
-| Nullroute Banned IPs | Change | Any | None
-| OS Troubleshooting | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |
-| Performance Issues | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |
-| Reset User Password on Active Directory Domain | Account / User Management | None | None |
-| Reset User Password on Server | Change | Server, VM | One |
-| Reset VPN User Password on Firewall | Change | Firewall Platform | One |
-| Colocation SmartHands and General Support Request | Change | Colocation | One |
-| User Password Reset/Account Unlock | Change | None | None |
-| Usage Drop | Change | Any | None |
-| Enroll in RPC-VMWare Guest OS Services | Change | Virtual Machine for Linux, Virtual Machine for Windows, VMware Service for OS Administration | One |
-| VPN - add, reset, delete VPN user(s) on firewall(s) | Account / User Management | Firewall Platform | One |
-| Website is Down | Change | Hypervisor, Hypervisor ESXi, Server, VM | One |
\ No newline at end of file
+| Common Request (Template Name) | Category | Product/Service | Allowed Device Selections | Minimum Devices Required |
+| ----------------------------- | -------- | ---------------- | ------------------- | ------------------------ |
+| Add Amor anywhere| Change | Dedicated Hosting | None | None |
+| Add CPU to Server or Hypervisor | Change  | Dedicated Hosting | Hypervisor, Hypervisor ESXi, Server, VM | One |                          
+| Add Disk Space to Server or Hypervisor | Change | Dedicated Hosting | Hypervisor, Hypervisor ESXi, Server, VM | One |  
+| Add RAM to Server or Hypervisor | Change | Dedicated Hosting | Hypervisor, Hypervisor ESXi, Server, VM | One |  
+| Create New User on Active Domain Directory | Account | User Management | None | None |
+| Create New User on Server | Change | Dedicated Hosting | Server, VM | One |
+| Add Virtual IP Address to Load Balancer | Change | Dedicated Hosting | Load Balancer | One |
+| Create new VPN User on Firewall | Change | Dedicated Hosting | Firewall Platform | One |
```

Comment:
Add the `On-site Physical Colocation Access Request` template after this line. it requires change > Dedicated Hosting > Colocation (device type selection).